src/js/generateLottos.js

```diff
@@ -0,0 +1,27 @@
+import { LOTTO_LENGTH, LOTTO_NUMBER_RANGE_MAX, LOTTO_NUMBER_RANGE_MIN } from './constants.js';
+
+const generateLottoNumbers = () => {
+  const lottoNumbers = Array.from({ length: LOTTO_NUMBER_RANGE_MAX }).map(
+    (_, i) => i + LOTTO_NUMBER_RANGE_MIN
+  );
+  return lottoNumbers
+    .sort(() => Math.random() - 0.5)
+    .slice(0, LOTTO_LENGTH)
+    .sort((a, b) => a - b);
+};
```

Comment:
```suggestion
  const lottoNumbers = Array.from({ length: LOTTO_NUMBER_RANGE_MAX }).map(
    (_, i) => i + LOTTO_NUMBER_RANGE_MIN
  );
  
const generateLottoNumbers = () => {
  return lottoNumbers
    .sort(() => Math.random() - 0.5)
    .slice(0, LOTTO_LENGTH)
    .sort((a, b) => a - b);
};
```
이렇게 1 ~ 45의 숫자는 아예 함수 밖에다 선언해서 사용해도 좋을 것 같아요!

Reply:
넵 감사합니다!

Reply:
@JunilHwang 
넵 해당 부분은 수정해서 step2 pr 반영본에 올리겠습니다!